libs/ng-utils/src/lib/use-http-image-source/use-http-image-source.pipe.ts

```diff
@@ -1,6 +1,7 @@
 import { HttpClient, HttpResponse } from '@angular/common/http';
 import { ChangeDetectorRef, Inject, OnDestroy, Pipe, PipeTransform } from '@angular/core';
 import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+import { isTruthy } from '@this-dot/utils';
```

Comment:
if these are internal-utils, I'd recommend renaming the package to utils-internal or something like that.